drill-yarn/src/test/java/org/apache/drill/yarn/core/TestConfig.java

```diff
@@ -81,7 +81,6 @@ protected URL findResource(String name) {
         try {
           return file.toURI().toURL();
         } catch (MalformedURLException e) {
-          ;
```

Comment:
Could you please add a comments like `// noop` to avoid warnings for empty catch statement?

Reply:
done